### ISSUE TITLE
Not loading image

### ISSUE THREAD

**Issue by DanielZanchi** [viSoom/WebP-Swift#5]
let imageWebP = UIImage(webpWithData: imagePNGData! as NSData)
Doesn't return an image 😟

**Comment by visoom** [comment]
Sorry for the inconvenience! 
I'm no longer maintaining this project, If I found the time to update the binaries I'll fix this issue! 

Marco

**Comment by visoom** [comment]
Hi Danny, I've updated the project to XCode 10, I included the Google framework and added an example of NSData WebP usage:

```
        //Simple usage example with NSData
        let filePath = Bundle.main.path(forResource: "3_webp_ll", ofType: "webp")!
        var fileData:NSData? = nil
        do {
            fileData = try NSData(contentsOfFile: filePath, options: NSData.ReadingOptions.uncached)
        }
        catch {
            print("Error loading WebP file")
        }
        let imageWithData = UIImage(webpWithData: fileData!)
```
I hope this will help you!

**Comment by DanielZanchi** [comment]
Ok, I will try this. 
I was getting the Data from a UIImage this way. 
Let image = UIImage(....)
Let imageData = image.pngData() as NSData
Let webpUIImage = UIImage(webpWithData: imageData!)

I believe this isn’t possible.
(Sorry if there’s sintax errors, I wrote from my iPhone)

**Comment by visoom** [comment]
Exactly, the purpose of this library is to transform WebP images (mostly from web) to an usable UIImage, you need  a converter from PNG to WebP!
It's possible to do it with the same linked Google framework but with other methods, If I have some time in the next days I'l try to implement it

**Comment by DanielZanchi** [comment]
I thought that I could use it for that purpose. I need to convert a UIImage with PNG to a WebP, what methods do you think I should use with the libwebp? 
Thanks for the help.
